tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -126,7 +126,7 @@ sub run_test {
     }
 
     if ($upload_virt_log_flag eq "yes") {
-        upload_virt_logs($log_dir, $compressed_log_name);
+        upload_virt_logs("$log_dir", $compressed_log_name);
```

Comment:
It can support passing in multiple directories, so as to be uploaded together as a compressed one.